### DISCUSSION TITLE
Page for the "Settings" to replace the Modal

### DISCUSSION THREAD

**Discussion by ismaail** [open-webui/open-webui#956] [Ideas]
More settings are added in new releases, and the Modal becomes crampy and it's hard to scroll to the setting i want to check or change.

Suggestion: add a dedicate page for the settings (like the page of Modelfiles, Prompts)

**Comment by justinh-rahb** [comment]
Agreed, I think you're right it is starting to get crowded. This would also fit better with future plans of other new "pages" that may be added later.
